scripts/build.py

```diff
@@ -127,7 +127,7 @@ def get_target_platforms():
 PACKAGES_CCTOOLS_PORT="cctools-port-darwin19-6c438753d2252274678d3e0839270045698c159b-linux"
 
 NODE_MODULE_LIB_DIR = os.path.join("ext", "lib", "node_modules")
-EMSCRIPTEN_VERSION_STR = "3.1.55"
+EMSCRIPTEN_VERSION_STR = "3.1.61"
```

Comment:
Is this the newest emscripten version? I think that if we have to update we should use the latest version since it needs to be packaged and stored on the extender (project build server) alongside the old versions of the SDK for backward compatability